srsue/src/main.cc

```diff
@@ -152,6 +152,7 @@ static int parse_args(all_args_t* args, int argc, char* argv[])
     ("rrc.mbms_service_port",   bpo::value<uint32_t>(&args->stack.rrc.mbms_service_port)->default_value(4321),                    "Port of the MBMS service")
     ("rrc.nr_measurement_pci",  bpo::value<uint32_t>(&args->stack.rrc_nr.sim_nr_meas_pci)->default_value(500),                    "NR PCI for the simulated NR measurement")
     ("rrc.nr_short_sn_support", bpo::value<bool>(&args->stack.rrc_nr.pdcp_short_sn_support)->default_value(true),                 "Announce PDCP short SN support")
+    ("rrc.sdu_fuzzed_bits",     bpo::value<int>(&args->stack.rrc_nr.sdu_fuzzed_bits)->default_value(0),                              "Number of bits to fuzz in the SDU buffer")
```

Comment:
Perhaps add "0 disables"?

Reply:
I'm not sure what you mean here, can you clarify? Thanks.